app/utils/biometricAuthentication.ts

```diff
@@ -0,0 +1,25 @@
+import { translate } from "../i18n"
+import FingerprintScanner from 'react-native-fingerprint-scanner'
+
+
+export const isSensorAvailable = async () => {
+  const telemetryType = await FingerprintScanner.isSensorAvailable()
+  return telemetryType !== null
+}
+
+export const authenticate = async (handleSuccess, handleFailure) => {
+  await FingerprintScanner.release()
+  FingerprintScanner
+    .authenticate({ 
+      description: translate("AuthenticationScreen.authenticationDescription"),
+      fallbackEnabled: true
+    })
+    .then(() => {
+      FingerprintScanner.release()
```

Comment:
could `FingerprintScanner.release()` be in a `finally` branch? or it needs to be called before `handleSuccess`/`handleFailure` function?

Reply:
Yes, this was me being unfamiliar/rusty with some parts of javascript.